core/jobs/batch_jobs/blog_validation_jobs.py

```diff
@@ -113,6 +113,27 @@ def run(
         )
 
 
+class FindDuplicateBlogAuthorDetailsModelForAuthorJob(base_jobs.JobBase):
+    """Validates that only one Blog Author Detail Models exists corresponding to
+    given author id.
+    """
+
+    def run(
+        self
+    ) -> beam.PCollection[blog_validation_errors.DuplicateblogAuthorModelError]:
+        return (
+            self.pipeline
+            | 'Get every Blog Author Details Model' >> (
+                ndb_io.GetModels(blog_models.BlogAuthorDetailsModel.query()))
+            | GetModelsWithDuplicatePropertyValues('author_id')
+            | 'Flatten models into a list of errors' >> beam.FlatMap(
+                lambda models: [
+                    blog_validation_errors.DuplicateblogAuthorModelError(model)
```

Comment:
"blog" should be capitalized in "DuplicateBlog"?

Reply:
Yes, it should be capitalized. Done